src/contractTest/java/uk/gov/hmcts/reform/idam/client/IdamClientConsumerTest.java

```diff
@@ -69,6 +106,24 @@ public RequestResponsePact executeGetIdamAccessTokenAndGet200(PactDslWithProvide
                 .toPact();
     }
 
+    @Test
+    @PactTestFor(pactMethod = "executeGetUserInfo")
+    void verifyUserInfo() {
+        UserInfo actualUserInfo = idamClient.getUserInfo("Bearer eyJ0eXAiOiJKV1QiLCJraWQiOiJiL082T3ZWdeRre");
```

Comment:
perhaps this token/header could be a constant

Reply:
Done.